### PR TITLE
CI(state): add linting to state and fix linting errors

### PR DESCRIPTION
**Pull request by lderrickable**
add linting to state and fix linting errors

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

Fixes: https://github.com/graycoreio/daffodil/issues/106

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: libs/state/src/product/containers/best-sellers/best-sellers.component.spec.ts [hunk 1]
```diff
@@ -64,10 +63,9 @@ describe('BestSellersContainer', () => {
     });
 
     it('sets bestSellers', () => {
-      combineLatest(
```
Comment (damienwebdev) [inline libs/state/src/product/containers/best-sellers/best-sellers.component.spec.ts]: Why was this changed?

Comment (lderrickable) [reply libs/state/src/product/containers/best-sellers/best-sellers.component.spec.ts]: It's depreciated

Comment (damienwebdev) [reply libs/state/src/product/containers/best-sellers/best-sellers.component.spec.ts]: @lderrickable kk.
